dynamorm/model.py

```diff
@@ -82,27 +82,21 @@ def should_transform(inner_class):
                 from .types._marshmallow import Schema
             elif "schematics" in sys.modules:
                 from .types._schematics import Schema
-
-                # Pull all of our fields up onto the main schema, obeying MRO
-                # This is done to ensure that "mixin" class fields get properly declared for Schematics
-                def pull_up_fields(cls):
-                    for base in reversed(cls.__bases__):
-                        for k, v in six.iteritems(base.__dict__):
-                            if isinstance(v, Schema.base_field_type()):
-                                setattr(attrs["Schema"], k, v)
-                        pull_up_fields(base)
-
-                pull_up_fields(attrs["Schema"])
             else:
                 raise DynaModelException(
                     "Unknown Schema definitions, we couldn't find any supported fields/types"
                 )
```

Comment:
This `should_transform` code seems pretty brittle as well - now that we need users to be explicit and leverage `base_schema_type`, can we remove this code that checks `sys.modules`?